acceptance-tests/src/test/java/uk/gov/di/test/acceptance/AccountManagementStepDefinitions.java

```diff
@@ -170,4 +171,16 @@ public void theExistingAccountManagementUserIsShownAnErrorMessage() {
         WebElement emailDescriptionDetails = driver.findElement(By.id("error-summary-title"));
         assertTrue(emailDescriptionDetails.isDisplayed());
     }
+
+    @When("the existing account management user enters their new mobile phone number with international code")
+    public void theExistingAccountManagementUserEntersTheirNewMobilePhoneNumberWithInternationalCode() {
+        WebElement passwordField = driver.findElement(By.id("phoneNumber"));
+        passwordField.sendKeys("+447944882875");
```

Comment:
We should probably inject this via an environment variable rather than hard coding it.

Reply:
Yeah, gonna add it as an env variable. Thanks!